### DISCUSSION TITLE
180 days Github account

### DISCUSSION THREAD

**Discussion by Alfnoc** [filecoin-project/community#199] [Mining]
Hello. How to get an 180 days old account

**Comment by rjan90** [comment]
You can request test-FIL for the calibration-net here: https://faucet.calibration.fildev.network/ ! There is no need to have an 180-day old Github-account to request test-FIL

**Comment by Alfnoc** [comment]
I need it for this
You need a GitHub account that is at least 180 days old. If you don't have a GitHub account that's old enough, get in touch with the team on Filecoin Slack (opens new window).

**Comment by pgs2001** [comment]
is there any chance to get somewhere (except faucet) tFIL's on hyperspace to commit 32gb sector?
otherwise it will take a lot of time...
> Batch 0:
> 	Error: pushing message to mpool: GasEstimateMessageGas error: message execution failed: exit SysErrInsufficientFunds(6), reason: message failed with backtrace:
> --> caused by: send::send -- sender does not have funds to transfer (balance 29.992259450376460795, transfer 13141.343562146206053708) (5: insufficient funds)
>
